caffe2/python/rnn_cell.py

```diff
@@ -234,6 +234,71 @@ def create_states(self, model):
         ]
 
 
+# based on http://pytorch.org/docs/master/nn.html#torch.nn.RNNCell
+class ElmanCell(RNNCell):
```

Comment:
Hm, I haven't seen it called Elman before - it's usually just "rnn" (which is taken) or "simple" or "basic".

Maybe BasicRNNCell then? (similar to TF)

Reply:
the pytorch docs describe it as "a multi-layer Elman RNN", but yeah I can switch to Basic